java-frontend/src/main/java/org/sonar/java/se/checks/CustomUnclosedResourcesCheck.java

```diff
@@ -50,10 +50,14 @@
 @NoSqale
 public class CustomUnclosedResourcesCheck extends SECheck {
 
-  enum Status {
-    OPENED, CLOSED
+  class Status {
+    Object OPENED = new Object();
+    Object CLOSED = new Object();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "CLOSED" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS00116)